### ISSUE TITLE
Feature Request: sync status indication or loading bar

### ISSUE THREAD

**Issue by jay4fusion** [astrada/google-drive-ocamlfuse#44]
I have a suggestion for a feature:
Some way of notifying the user of the current sync status (like % complete) so the user knows when all of the files are finished syncing.

I have > 20 GB on my google drive, and it includes data and figures that are large files. When I am working on my linux machine it would be nice to know when all my changes on my other machines are available

Thank you for your time! I really enjoy google-drive-ocamlfuse! It really helps me out :)

**Comment by dairefagan** [comment]
I know this is a very old feature request, but I wanted to +1 it.

I am currently relying on monitoring ocamlfuse's bandwidth usage with nethogs to determine when it has finished uploading all of the files.

**Comment by ericdallo** [comment]
yeah, a status of the current files being uploaded/download would be enough I think
